.github/workflows/test.yml

```diff
@@ -53,6 +53,22 @@ jobs:
     runs-on: ${{ matrix.os }}
 
     steps:
+# https://gist.github.com/alexellis/1f33e581c75e11e161fe613c46180771#file-metering-gha-md
+# vmmeter start
+    - uses: alexellis/arkade-get@master
+      if: matrix.os == 'actuated-arm64-4cpu-12gb'
```

Comment:
If you're interested - the metering also works on hosted runners.

Reply:
Thanks, but probably we do not need this for hosted runners, as we can't change the instance type for hosted runners anyway